frontend/src/components/VersionSelector/VersionSelector.tsx

```diff
@@ -0,0 +1,117 @@
+import { Button, Outline, P } from "@maykin-ui/admin-ui";
+import { FC, ReactNode, useState } from "react";
+
+import "./VersionSelector.css";
+
+type Version = {
```

Comment:
Can you match this to the API schema from the backend endpoint, and then move it to types.d.ts?